chainer_pytorch_migration/parameter.py

```diff
@@ -70,6 +73,12 @@ def __as_tensor(self, value):
             return _ChainerTensor(value)
         return value
 
+    def to(self, *, device):
+        if device is None:
+            return self
+        device = cpm.to_chainer_device(torch.device(device))
+        return LinkAsTorchModel(self.link, _device=device)
```

Comment:
`to` should modify and return `self`, instead of creating a new instance.

Reply:
I'm trying to make a fix for this comment, but it seems that it will not be resolved soon. So, I dropped the commit for `LinkAsTorchModel.to` and will support it in another PR.

Reply:
Then, how about implementing the reverse conversion for completeness?

Reply:
@niboshi Added `to_torch_device`. PTAL.